insights/core/plugins.py

```diff
@@ -191,6 +191,23 @@ def make_skip(rule_fqdn, reason, details=None):
             "type": "skip"}
 
 
+def is_make_reponse_too_long(key, kwargs):
```

Comment:
Nitpick: would you mind to add a leading underscore to the function name? which indicates it's an internal but not global interface.